tools/sdk-testgen/ci.yml

```diff
@@ -101,15 +103,15 @@ stages:
           steps:
           - template: ../../eng/pipelines/templates/steps/sdk-testgen-set-env.yml
```

Comment:
Release should use the common JS release template, https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/stages/archetype-sdk-publish-js.yml

Reply:
Updated to use the common pipeline template.